### PR TITLE
fix(install): fix start script with sentry config file generated

### PR DESCRIPTION
**Pull request by JonLazariniDD**
## Ticket title
One line description
[link to ticket number]()

## Checklist
- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,7 +72,7 @@
     "angular2-signaturepad": "^2.8.0",
     "audit-filter": "^0.5.0",
     "call-number": "^1.0.1",
-    "cordova-ios": "^5.0.1",
+    "cordova-ios": "5.1.1",
```
Comment (alex-deloitte) [inline package.json:75]: Have you tested this? This could potentially be a breaking change

File: package.json [hunk 2]
```diff
@@ -169,7 +172,9 @@
       "cordova-plugin-secure-storage": {},
       "cordova-plugin-app-version": {},
       "cordova-plugin-exit": {},
-      "sentry-cordova": {}
+      "sentry-cordova": {
+        "SENTRY_ANDROID_SDK_VERSION": "1+"
```
Comment (alex-deloitte) [inline package.json:176]: Why does it need Android SDK?

File: package.json [hunk 3]
```diff
@@ -182,4 +187,4 @@
       "pre-push": "npm run build && npm run test-coverage"
     }
   }
-}
+}
```
Comment (alleypa) [inline package.json:190]: remove line terminal

**Comment by alleypa** [comment]
also squash

**Review by alex-deloitte — APPROVED** [review]

**Review by alleypa — APPROVED** [review]

**Pull request merged**